webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.scss

```
@@ -1,4 +1,4 @@
-@import '~@theforeman/vendor/scss/variables';
+@import '../../common/variables.scss';
```

```suggestion
@import '../../common/variables';
```
I don't think it's required